src/js/views/globals/search/patient-search.scss

```diff
@@ -15,16 +15,41 @@
   }
 }
 
+.patient-search__clear-type-button {
+  background-color: color(light_blue, light);
+  border: none;
+  color: $black-20;
+  font-size: 14px;
+  height: 28px;
+  margin-right: 8px;
+  padding: 4px 12px;
+
+  .patient-search__clear-type-button-label {
+    vertical-align: middle;
+  }
+
+  .icon {
+    font-size: 14px;
+    margin-left: 8px;
+    vertical-align: bottom;
+    width: 1em;
+  }
+}
+
 .patient-search__input {
   border: none;
   display: flex;
+  flex-grow: 1; // tells <input /> to fill width regardless of button size
   font-size: 24px;
   font-weight: normal;
 
   // height = line-height: 1.2
   height: 28px;
   padding-right: 24px;
-  width: 100%;
+
+  &[type='date']::-webkit-calendar-picker-indicator {
+    display: none;
+  }
```

Comment:
Can we do this instead of doing the overflow we mentioned on Slack?

This should hide the calendar icon, but still allow the date picker to open on mobile/iOS.

Reply:
only works on chrome